### PR TITLE
added session 9 hw

### PR DESCRIPTION
**Pull request by Kielv2**

### CODE REVIEW

File: ASPNetCoreMasters/Controllers/UsersController.cs [hunk 1]
```diff
@@ -107,8 +107,9 @@ private async Task<string> GenerateTokenAsync(IdentityUser user)
         {
             IList<Claim> userClaims = new List<Claim>
             {
-            new Claim(ClaimTypes.Email, user.Email),
-            new Claim(ClaimTypes.Name, user.UserName)
+                new Claim(ClaimTypes.Email, user.Email),
+                new Claim(ClaimTypes.Name, user.UserName),
+                new Claim(ClaimTypes.NameIdentifier, user.Id.ToString())
```
Comment (kevinatgithub) [inline ASPNetCoreMasters/Controllers/UsersController.cs:112]: _jwtOptions is null

File: ASPNetCoreMasters/Controllers/UsersController.cs [hunk 2]
```diff
@@ -107,8 +107,9 @@ private async Task<string> GenerateTokenAsync(IdentityUser user)
         {
             IList<Claim> userClaims = new List<Claim>
             {
-            new Claim(ClaimTypes.Email, user.Email),
-            new Claim(ClaimTypes.Name, user.UserName)
+                new Claim(ClaimTypes.Email, user.Email),
+                new Claim(ClaimTypes.Name, user.UserName),
```
Comment (kevinatgithub) [inline ASPNetCoreMasters/Controllers/UsersController.cs:111]: Also, can you make the login endpoint to HttpPost? when using HttpGet, we might be exposing the password of user in the query string/

**Review by kevinatgithub — CHANGES_REQUESTED** [review]

File: ASPNetCoreMasters/Controllers/UsersController.cs (hunk 2 above)
Comment (Kielv2) [reply ASPNetCoreMasters/Controllers/UsersController.cs:111]: changed Get to post

File: ASPNetCoreMasters/Controllers/UsersController.cs (hunk 1 above)
Comment (Kielv2) [reply ASPNetCoreMasters/Controllers/UsersController.cs:112]: changed UsersController constructor

**Pull request merged**